torch/_export/__init__.py

```diff
@@ -433,24 +484,37 @@ def to_str_dict(sig_component: Dict[Any, Any]):
         export_graph_signature,
         flat_args,
     )
+
+    if isinstance(f, torch.nn.Module):
+        _replace_param_buffer_names(param_buffer_table, export_graph_signature)
+
+    module_call_signatures = {fqn: ModuleCallSignature(inputs=[], outputs=[], **specs) for fqn, specs in module_call_specs.items()}
+
+    if len(preserve_module_call_signature) > 0:
+        gm = CollectTracepointsPass(module_call_signatures)(gm).graph_module
+
     assert orig_out_spec is not None
     exported_program = ExportedProgram(
         gm,
         gm.graph,
         export_graph_signature,
+        # TODO(zhxchen17) Remove this field.
         CallSpec(in_spec, orig_out_spec),
-        params_buffers,
+        # TODO(zhxchen17) Return empty state_dict for functions.
+        {param_buffer_table.get(name, name): tensor for name, tensor in params_buffers.items()}
+        if isinstance(f, torch.nn.Module) else param_buffer_table,
         range_constraints,
         equality_constraints,
+        [ModuleCallEntry("", ModuleCallSignature(inputs=[], outputs=[], in_spec=orig_in_spec, out_spec=orig_out_spec))] +
         [ModuleCallEntry(fqn, sig) for fqn, sig in module_call_signatures.items()],
         (args, {}),
     )
 
-    exported_program = exported_program._transform(
-        _AddRuntimeAssertionsForInlineConstraintsPass(range_constraints, equality_constraints)
-    )
-    if len(preserve_module_call_signature) > 0:
-        exported_program = exported_program._transform(CollectTracepointsPass(module_call_signatures))
+    if len(range_constraints) > 0 or len(equality_constraints) > 0:
```

Comment:
Do you know if we put the unbacked symint based ranges into above two lists?

Reply:
I think range_constraints is for both unbacked and backed symints?

Reply:
Yeah thanks for confirming!